input/vulnerabilities/Qualcomm_TrustZone.json

```diff
@@ -10,7 +10,7 @@
   "Reported_on" : [["2015-10-13", "trustzone-blog"]],
   "Fixed_on" : [["2016-05-02", "trustzone-blog"]],
   "Fix_released_on" : [["2016-05-02", "trustzone-blog"]],
-  "Affected_versions" : [],
+  "Affected_versions" : [["6.0.1", "cvedetails-qualcomm-trustzone"]],
```

Comment:
Only the one minor version? We need a regular expression as well.

Reply:
Yes, according to https://www.cvedetails.com/cve/CVE-2016-2431, although this does seem rather odd